pennylane_lightning/core/src/lightning_gpu.toml

```diff
@@ -0,0 +1,117 @@
+schema = 1
+
+[device]
+name = "lightning.qubit"
+precision = ['float32', 'float64']
+
+[operations]
+observables = [
+	"NamedObs",
+	"HermitianObs",
+	"TensorObs",
+	"HamiltonianObs",
+]
+
+# These are gates which are defined in Quantum Instruction Set.
+[[operations.gates]]
+native = [
+        "QubitUnitary",
+        "PauliX",
+        "PauliY",
+        "PauliZ",
+        "MultiRZ",
+        "Hadamard",
+        "S",
+        "T",
+        "CNOT",
+        "SWAP",
+        "CSWAP",
+        "Toffoli",
+        "CY",
+        "CZ",
+        "PhaseShift",
+        "ControlledPhaseShift",
+        "RX",
+        "RY",
+        "RZ",
+        "Rot",
+        "CRX",
+        "CRY",
+        "CRZ",
+        "CRot",
+        "IsingXX",
+        "IsingYY",
+        "IsingZZ",
+        "IsingXY",
+        # Adjoint'ed gates
+        # which just need the inverse flag set
+        # are considered natively supported gates.
+        "Adjoint(S)",
+        "Adjoint(T)",
+]
+# decomp gates are those which are neither supported natively
+# and are not executed via the applyMatrix operation. Technically
+# they could also be decomposed via applyMatrix.
+decomp = [
+        "SX",
+        "ISWAP",
+        "PSWAP",
+        "SISWAP",
+        "SQISW",
+        "CPhase",
+        # Adjoint'ed gates
+        # which are not in native set must either be decomposed
+        # or applied via matrix.
+        "Adjoint(SX)",
+        "Adjoint(ISWAP)",
+        "Adjoint(SISWAP)",
+]
+# QubitUnitary should never be here.
+# Gates in "matrix" are decomposed to QubitUnitary.
+# Having a QubitUnitary here would mean an infinite loop.
+
+# These are gates that appear on the list of supported gates
+# but are not in the list of Quantum Gate Set Instructions in
+# the Catalyst repository.
+# TODO: Move some of these to native once we add support for them
+# in Catalyst runtime.
+matrix = [
+        "Identity",
+        "BasisState",
+        "QubitStateVector",
+        "StatePrep",
+        "ControlledQubitUnitary",
+        "MultiControlledX",
+        "DiagonalQubitUnitary",
+        "SingleExcitation",
```

Comment:
I mentioned that in the TODO. We will move them out of here once we have `__qir__qis__SingleExcitation` and the like. Because otherwise, they would be translated to these functions that are unimplemented.